app/models/generic_file.rb

```diff
@@ -12,4 +12,9 @@ class GenericFile < ActiveFedora::Base
   property :work_name, predicate: ::RDF::URI('http://docs.osfashland.org/terms/work_name'), multiple: false do |index|
     index.as :stored_searchable, :facetable
   end
+
+  # override sufia property
+  property :resource_type, predicate: ::RDF::DC.type, multiple: false do |index|
```

Comment:
This doesn't seem like it is working, and with the ugly hack to the mixin, won't be needed.